lib/helpers.js

```diff
@@ -182,10 +182,7 @@ helpers.isHyphenatedBEM = function (str) {
 };
 
 helpers.isValidHex = function (str) {
-  if (str.match(/^([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/)) {
-    return true;
-  }
-  return false;
+  return /^([a-f0-9]{8}|[a-f0-9]{6}|[a-f0-9]{4}|[a-f0-9]{3})$/i.test(str);
```

Comment:
```suggestion
  return /^([A-Fa-f0-9]{8}|[A-Fa-f0-9]{6}|[A-Fa-f0-9]{4}|[A-Fa-f0-9]{3})$/i.test(str);
```
Your pattern did not match against hex values containing uppercase letters.